homeassistant/components/assist_pipeline/pipeline.py

```diff
@@ -1325,6 +1335,12 @@ async def execute(self) -> None:
         stt_audio_buffer: list[ProcessedAudioChunk] = []
         stt_processed_stream: AsyncIterable[ProcessedAudioChunk] | None = None
 
+        # Timeout before another pipeline with the same wake word phrase can run again
+        cooldown_seconds: float = DEFAULT_WAKE_WORD_COOLDOWN
+        if self.run.wake_word_settings is not None:
+            # Use cooldown from wake settings if available
+            cooldown_seconds = self.run.wake_word_settings.cooldown_seconds
```

Comment:
It feels weird that voice satellite integrations can override this global cooldown.

Reply:
Especially since it's on a per-call basis. So if ESPHome comes in, it's 5s but when Wyoming comes in, it could be 10s ?

Reply:
Yeah, that's a good point. I will just make it a shared default value.

Reply:
This is now a shared constant.